documentation_quality_analysis/analyze_library/prioritize_api/llm/discusses_api_pipeline.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+from string import Template
+
+import pandas as pd
+from bs4 import BeautifulSoup
+
+from analyze_library.prioritize_api.heurictics.has_api_analysis import get_all_posts
+from analyze_library.prioritize_api.llm.detect_api_discussion import detect_api_discussion
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+TEMPLATE_PATH = os.path.join(ROOT_DIR, "templates")
+DATA_PATH = os.path.join(ROOT_DIR, "data")
+
+
+def get_prompt_messages(library, api_fqn, post_body, mode='one_shot'):
+    metadata = {
+        'library': library,
+        'api_fqn': api_fqn,
+        'post_body': post_body
+    }
+
+    with open(os.path.join(DATA_PATH, "discusses_api_example.txt")) as f:
+        metadata['learning_example'] = f.read()
```

Comment:
this is currently an empty file?

Reply:
Yes, I didn't use this yet, this will change as we decided to keep the prompt and the template separate.